plugins/mcreator-localization/lang/texts.properties

```diff
@@ -508,6 +508,10 @@ blockly.block.entity_isinvisible=Is %1 invisible
 blockly.block.entity_isinvulnerable=Is %1 invulnerable
 blockly.block.entity_isalive=Is %1 alive
 blockly.block.entity_isnonboss=Is %1 non-boss
+blockly.block.entity_isinwater=Is %1 in water
+blockly.block.entity_isinlava=Is %1 in lava
+blockly.block.entity_isinwater_orbubblecolumn=Is %1 in water or bubble column
+blockly.block.entity_isinwaterrain_orbubblecolumn=Is %1 in water rain or bubble column

```

Comment:
Commas are missing here:
Is %1 in water, rain, or bubble column

Reply:
Done!